<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analizador Sintáctico</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/new/prefix1/img/Icon.png</normaloff>:/new/prefix1/img/Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 69, 138);</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 69, 138);
alternate-background-color: rgb(40, 69, 138);
border-color: rgb(40, 69, 138);
gridline-color: rgb(40, 69, 138);
selection-color: rgb(40, 69, 138);
selection-background-color: rgb(40, 69, 138);</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabEditor">
     <attribute name="title">
      <string>Editor</string>
     </attribute>
     <widget class="QPlainTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>621</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
selection-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAnalizar">
      <property name="geometry">
       <rect>
        <x>151</x>
        <y>418</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Analizar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTablas">
     <attribute name="title">
      <string>Tablas</string>
     </attribute>
     <widget class="QTableWidget" name="tableL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>291</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(188, 188, 188);
color: rgb(0, 0, 0);
selection-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Analizador Léxico</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableS">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>291</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(188, 188, 188);
color: rgb(0, 0, 0);
selection-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>191</width>
        <height>57</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Analizador Sintáctico</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>(Producciones)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="lblResultado">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>440</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SINTAXIS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>?</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>10</y>
        <width>621</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato Black</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Instituto Tecnológico de la Laguna</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>456</width>
        <height>85</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Lato Black</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>App Analizador Sintáctico GUI</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Lato Black</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Lenguajes y Autómatas l</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>426</width>
        <height>85</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Lato Black</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Desarrollado por:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Lato Black</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Iván Romero Canaán 18131278</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>360</y>
        <width>251</width>
        <height>85</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Lato Black</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Semestre:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Lato Black</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Enero - Junio 2021</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato Black</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="font">
     <font>
      <family>Lato Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Menú</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionReiniciar"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/new/prefix1/img/abrir2.png</normaloff>:/new/prefix1/img/abrir2.png</iconset>
   </property>
   <property name="text">
    <string>Abrir archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/new/prefix1/img/guardar.png</normaloff>:/new/prefix1/img/guardar.png</iconset>
   </property>
   <property name="text">
    <string>Guardar archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/new/prefix1/img/salir.png</normaloff>:/new/prefix1/img/salir.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReiniciar">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/new/prefix1/img/reiniciar.png</normaloff>:/new/prefix1/img/reiniciar.png</iconset>
   </property>
   <property name="text">
    <string>Reiniciar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
